prediction_market_agent/tools/custom_crewai_tools.py

```diff
@@ -1,3 +1,7 @@
+from prediction_market_agent.utils import patch_sqlite3
+
+patch_sqlite3()
```

Comment:
Should this not be in the streamlit app instead of in the tools?

Reply:
Yes, that makes more sense! Moved.